src/site/content/en/blog/backdrop-filter/index.md

```diff
@@ -0,0 +1,185 @@
+
+Design techniques and styles previously reserved for native operating systems are now performant and achievable with a single CSS declaration. Let's look at some examples.
+
+### Single filter
+
+In the following example, the frosted effect is achieved by combining color and blur. The blur is supplied by `backdrop-filter`, while the color comes from the element's semi-transparent background color.
+
+```css
+.blur-behind-me {
+  background-color: rgba(255, 255, 255, 0.3);
+  backdrop-filter: blur(.5rem);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-rgb.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Try this example for yourself in <a href="https://codepen.io/netsi1964/pen/JqBLPK" target="_blank">CodePen</a>.
+  </figcaption>
+</figure>
+
+### Multiple filters
+
+Sometimes you'll need multiple filters to achieve the desired effect. To do this, provide a list of filters separated by a space. For example:
+
+```css
+.brighten-saturate-and-blur-behind-me {
+  backdrop-filter: brightness(150%) saturate(150%) blur(1rem);
+}
+```
+
+In the following example, each of the four panes have different combinations of backdrop filters while the same set of shapes are animated behind them.
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-ambient_blur.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Try this example for yourself in <a href="https://codepen.io/pepf/pen/GqZkdj" target="_blank">CodePen</a>.
+  </figcaption>
+</figure>
+
+### Overlays
+
+This example shows how to blur a semi-transparent background to make text readable while stylistically blending with a page's background.
+
+```css
+.modal {
+  backdrop-filter: blur(10px);
+  background-color: rgba(255, 255, 255, 0.5);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-modal.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Try this <a href="https://mfreed7.github.io/backdrop-filter-feature/examples/scrollable.html" target="_blank">example</a> for yourself.
+  </figcaption>
+</figure>
+
+### Text contrast on dynamic backgrounds
+
+I said earlier that `backdrop-filter` allows performant effects that would be difficult or impossible on the web. An example of this is changing a background in respone to an animation. In this example, `backdrop-filter` maintains the high contrast between the text and its background in spite of what's going on behind the text. It starts with the default background color `darkslategray` and uses `backdrop-filter` to invert the colors after the transformation.
```

Comment:
web.dev avoids first person singular. If the revision proposed in the first review doesn't feel good, feel free to go with something else!